src/index.ts

```diff
@@ -149,10 +149,10 @@ export default (opts: PluginOptions = {}): Plugin => {
         }
       })
     },
-    async resolveId(id, importer) {
+    async resolveId(id, importer, options) {
       if (importer && !relativeImportRE.test(id) && !isAbsolute(id)) {
         const viteResolve: ViteResolve = async (id, importer) =>
-          (await this.resolve(id, importer, { skipSelf: true }))?.id
+          (await this.resolve(id, importer, options))?.id
```

Comment:
It still needs `skipSelf: true` I think?

Reply:
From the [Vite docs for `this.resolve`](https://rollupjs.org/plugin-development/#this-resolve):

> The default of `skipSelf` is `true`

Reply:
Oh nice. That's a welcome change from Rollup :)

Reply:
Isn't this a breaking changes for people still using Vite 3-4?
Rollup changed the default into v4, which is used by Vite 5, thus this is a breaking change and would need a proper major release

Reply:
@IlCallo Ah good point! I think I'll explicitly set `skipSelf: true` for backwards compatibility.

Reply:
Fixed in v4.2.3

Reply:
Thanks 😁